nss-day-cohort-44/rare-rest-api-malicous-vortices
# Dd comments

# ISSUE TICKET 
## Description
Added a README and commented some of the trickier code I wrote

## Type of change

# Testing Instructions
`git fetch origin `
`git checkout ` 

Just make sure everything still works

User Instructions: 

# Checklist:
